client/blocks/login/lost-password-form.jsx

```diff
@@ -1,4 +1,4 @@
-import { FormInputValidation } from '@automattic/components';
+import { FormInputValidation, Spinner } from '@automattic/components';
```

Comment:
Hey @ilyasfoo I think we should use `Spinner` from `@wordpress/components` (Y5pUYSJPsGEud1vknUZhi8-fi-1749_35798). 🙂